src/components/AssignedMembers/AssignedMembersContainer.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { useStripes } from '@folio/stripes/core';
+import {
+  Headline,
+  Badge,
+  Loading,
+  Accordion,
+} from '@folio/stripes/components';
+
+import useAssignedUsers from './hooks/useAssignedUsers';
+import AssignedMembersList from './AssignedMembersList';
+
+const AssignedMembersContainer = ({ permissionsSet, expanded, onToggle }) => {
+  const stripes = useStripes();
+  const tenantId = stripes.okapi.tenant;
```

Comment:
Is it enough to reuse this component in the `ui-consortia-settings`? It seems to me that you will need the ability to explicitly specify for which tenant to load the list.

Reply:
You mean, the `tenantId` should code from `props`, right?

Reply:
yes, otherwise you won't be able to specify tenant in consortium manager